### PR TITLE
API provides functionality to just let authorized staff members create articles

### PR DESCRIPTION
**Pull request by kfrostare**
# [PT Story](https://www.pivotaltracker.com/story/show/171913119)
```
As a journalist,
In order to restrict who can write articles
I would like to be required to authorize myself when creating articles
```
## Changes proposed in this pull request:
* Made unit test for new features in the controller
* Updates create method in controller
* Creates private method for user role

## What I have learned working on this feature:
* Repetition is golden!
* Refreshed knowledge of testing environment in rails using different headers

### CODE REVIEW

File: app/controllers/api/articles_controller.rb [hunk 1]
```diff
@@ -16,13 +17,18 @@ def show
     render json: article, status: 200
   end
 
-  def create
-    article = Article.create(article_params)
-    if article.persisted?
-      render json: { message: 'Your article is ready for review.' }, status: 200
+  def create #Needs refactoring
```
Comment (hmvitous) [inline app/controllers/api/articles_controller.rb]: Remove commented code

**Review by hmvitous — COMMENTED** [review]
Looks good to me. 
Just make sure the commented code is taken out.

File: spec/requests/api/client_can_login_spec.rb [hunk 2]
```diff
@@ -27,7 +28,7 @@
     end
 
     it 'returns 200 response status' do
-      expect(response). to have_http_status 200
+      expect(response).to have_http_status 200
```
Comment (johanbounce) [inline spec/requests/api/client_can_login_spec.rb:31]: good spot!

**Comment by johanbounce** [comment]
Looks fine if comments are resolved

**Pull request merged**